service/src/main/java/com/derpgroup/derpwizard/resource/AlexaResource.java

```diff
@@ -65,75 +67,28 @@ public AlexaResource(MainConfig config, Environment env) {
    * @return The message, never null
    */
   @POST
-  public SpeechletResponseEnvelope doAlexaRequest(SpeechletRequestEnvelope request){
-    if(request == null || request.getRequest() == null){
+  public SpeechletResponseEnvelope doAlexaRequest(@NotNull @Valid SpeechletRequestEnvelope request){
```

Comment:
Adding the `@NotNull` annotation allows us to logically remove the null-check on the `request` object on line 69. Adding the annotation is a little more human-readable than implying that the framework will handle the null-check for us, but I can take it out.

I added the `@Valid` annotation optimistically in case they implement bean validation in a future version. I can remove it as well.